packages/react-next/src/components/Slider/Slider.test.tsx

```diff
@@ -220,87 +149,7 @@ describe('Slider', () => {
     }, ONKEYDOWN_TIMEOUT_DURATION);
 
     jest.runOnlyPendingTimers();
-  });
-
-  it('does not update the value when slider is controlled', () => {
```

Comment:
It looks like a lot of these test have been removed, did they not work? Should they be removed?